automations/winchautomation.py

```diff
@@ -8,29 +8,29 @@ class WinchAutomation(StateMachine):
     sd = NetworkTable
 
     @state(first=True)
-    def initialise_winch(self):
+    def catch_rope(self):
         self.put_dashboard()
-
         self.winch.piston_open()
         self.winch.rotate_winch(0.3)
-        self.next_state("climb")
+
+        if self.winch.on_rope_engaged() and state_tm > 2:
```

Comment:
Shouldn't `state_tm` be listed as an argument for this state?

As an aside, we should probably have tests for the automations to catch these bugs…